yarn.lock

```diff
@@ -2369,6 +2378,70 @@
     "@types/yargs" "^15.0.0"
     chalk "^4.0.0"
 
+"@material-ui/core@^4.11.0":
```

Comment:
the yarn lock still contains material ui references.. delete yarn lock(this file), then make sure that the packages are correct and rebuild this lock file by calling "yarn install"